micro-rdk/src/common/grpc.rs

```diff
@@ -1212,6 +1213,16 @@ where
         self.encode_message(operation)
     }
 
+    // robot_shutdown will not return anything because will restart
+    fn robot_shutdown(&mut self, _: &[u8]) -> ! {
```

Comment:
You could just not take an argument here too. Nothing requires you to pass `payload` above, that I see.

Reply:
I see some other requests that don't take any arguments still take a payload and don't use the value, like get_operations(), Resource_names(), etc... Is shutdown a different situation?

Reply:
I wonder if you get an unused warning above or something if you do that. OK, nevermind, just follow the pattern that is in use and go with what you have here now.